index.html

```diff
@@ -148,7 +162,7 @@ <h1 class="span12 topic">Download &amp; how to use in your application</h1>
         <p>
           There are multiple ways to download and use GeoExt. GeoExt can be used
```

Comment:
This should be reworded to include a npm i as first option IMO

Reply:
I've added the install options as a list and included npm